packages/block-library/src/tag-cloud/edit.js

```diff
@@ -82,6 +89,20 @@ function TagCloudEdit( { attributes, setAttributes, taxonomies } ) {
 					max={ MAX_TAGS }
 					required
 				/>
+				<NumberControl
+					label={ __( 'Smallest Font Size (Pt)' ) }
+					value={ smallestFontSize }
+					onChange={ ( value ) =>
+						setAttributes( { smallestFontSize: value } )
+					}
+				/>
+				<NumberControl
+					label={ __( 'Largeest Font Size (Pt)' ) }
```

Comment:
I guess we can make labels smaller like `Largest size(pt)`. Also note the typo in `Largest` here.